content_script.js

```diff
@@ -95,34 +97,34 @@ $(function() {
-
-          ics_content = "BEGIN:VEVENT\n"
-                      + "DTSTART:"  + formatDateTime(date_before, start_time) + "\n"
-                      + "DTEND:"    + formatDateTime(date_before, end_time) +"\n"
-                      + "LOCATION:" + room + "\n"
-                      + "RRULE:FREQ=WEEKLY;UNTIL=" + formatDateTime(end_date, end_time) + ";BYDAY=" + days_of_week + "\n"
-                      + "EXDATE:"   + formatDateTime(date_before, start_time) + "\n"
-                      + "SUMMARY:"  + "(" + component + ") " + event_name + "\n"
-                      + "DESCRIPTION:"
-                        + 'Section: '         + section + '\\n'
-                        + 'Instructor: '      + instructor + '\\n'
-                        + 'Component: '       + component + '\\n'
-                        + 'Class Number: '    + class_number + '\\n'
-                        + 'Days/Times: '      + days_times + '\\n'
-                        + 'Start/End Date: '  + start_end_date + '\\n'
-                        + 'Location: '        + room + '\\n\n'
-                      + "END:VEVENT\n";
+          start_time    = getTimeString(start_end_times[0]);
+          end_time      = getTimeString(start_end_times[1]);
+
+          days_of_week  = getDaysOfWeek(days_times.match(/[A-Za-z]* /)[0])
+
+          ics_content = "BEGIN:VEVENT\n" +
+                        "DTSTART:" + formatDateTime(date_before, start_time) + "\n" +
+                        "DTEND:" + formatDateTime(date_before, end_time) + "\n" +
+                        "LOCATION:" + room + "\n" +
+                        "RRULE:FREQ=WEEKLY;UNTIL=" + formatDateTime(end_date, end_time) + ";BYDAY=" + days_of_week + "\n" +
+                        "EXDATE:"   + formatDateTime(date_before, start_time) + "\n" +
+                        "SUMMARY:"  + "(" + component + ") " + event_name + "\n" +
+                        "DESCRIPTION:" +
+                          'Section: '        + section + '\\n' +
+                          'Instructor: '     + instructor + '\\n' +
+                          'Component: '      + component + '\\n' +
+                          'Class Number: '   + class_number + '\\n' +
+                          'Days/Times: '     + days_times + '\\n' +
+                          'Start/End Date: ' + start_end_date + '\\n' +
+                          'Location: '       + room + '\\n\n' +
+                        "END:VEVENT\n";
 
           ics_content_array.push(ics_content);
 
           $(this).find('span[id*="MTG_DATES"]').append(
-            '<a href="#" onclick="window.open( \'data:text/calendar;charset=utf8,'
-              + escape( ics_content_wrap(ics_content) )
-              + '\' );"><div>ICS file</div></a>'
+            '<a href="#" onclick="window.open(\'data:text/calendar;charset=utf8,' +
+            escape(ics_content_wrap(ics_content)) +
+            '\');"><div>Download Schedule</div></a>'
```

Comment:
I've been meaning to change this - make this `...>Download Class</a>` (remove `<div>`)